### DISCUSSION TITLE
just a query

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#845] [General]
just out of curiosity i want to know how the touchpad gestures work ? i swipe with three fingers and desktop changes . what is he package and where is the config file ? TIA!

**Comment by kRHYME7** [comment]
https://github.com/prasanthrangan/hyprdots/blob/main/Configs%2F.config%2Fhypr%2Fhyprland.conf#L88-L91

**Comment by sudobrat** [comment]
can i add 4 finger control for changing spotify tracks and also volume in both directions ?? Thanks in advance:)

**Reply by kRHYME7** [reply]
Hi, I don't think It is supported yet on hyprland alone.

Here's the things you can do for gestures. 
https://wiki.hyprland.org/Configuring/Variables/#gestures

but you can use libinput you can follow this discussion:
https://www.reddit.com/r/hyprland/comments/12qjea7/comment/k6nqpyk/
here's the dots
https://github.com/eispalast/dotfiles/blob/main/.config/libinput-gestures.conf


Here's the the libinput repo 
https://github.com/bulletmark/libinput-gestures

Here's the libinput-gesture for arch 
https://aur.archlinux.org/packages/libinput-gestures
